.travis.yml

```diff
@@ -5,6 +5,11 @@ dist: trusty
 jdk:
   - oraclejdk8
 
+env:
+  global:
+  - secure: FqsKkQflWF2MdS9r7fFOzUlX6Vj9Mua9rh/u359EmKWY77G24WgqAChtenl9gKQ2THqSMdWaKcuLLe+XvVBJtTGpVluhIXKiEgy712pURbpRizZfuOC2m/0Cr+MebXFt088VelHj1w8Xzg1pdZLEM/7GAGCHkrxJgv11CAhEzYmSl+NZi57Wuf9V1KYginSkj6/OcVz/GcmFi3MPJT2wUhw2SfXM/syCEQ54Cp//8F4UCCq39ZITuVzVjhqI3GOJuqxHN6kWh+mtTJqmbddVUPWmSyFKeaTWemn2OivYFdCbOD/NfrSZbYJKUSnPUFS5I388gboHoKIk1Kj2YyEbXG+cmOtoP0dGiV617wOd1CJnBperQgcj2/NUs70hyxBRn970NTYTz4ARwjAlKE+RQyeWIPZqEKXW5+Al0DFEhxKPGduMAia3iVomJrTwGtHw/r5tNQwwa+sRbIjbm46e8YeUrrgQWlPx0IBR8ON00yIfxBwfReh6SxJ87A+Iqe+ovHNhcDwIn0slz8VyAhHdb5j/Amdmp0UDk4BxS+SX7lhB4EmNs5oG+7kKXkCbZ14rUMMXKCBhZkd4CxV7BvK0hr5Nzhh3fQlC9Fsl757mCsXVtM6sp/PYRwLA7N0/X38Kvd1ON3Y2uLuNSZji93PdZTDTy5zsvC4K5OFpodz0fwI=
```

Comment:
Is it safe to have these here?

Reply:
https://docs.travis-ci.com/user/encryption-keys/